### DISCUSSION TITLE
Emission at runtime

### DISCUSSION THREAD

**Discussion by artyomabgaryan** [atteneder/glTFast#546] [Q&A] [answered]
Emission parameter cs doesn't work at runtime if emission texture was not set in glb.
how to fix this or is there any workarounds ?

**Comment by atteneder** [comment, marked as answer]
Hi,
This is obviously a bug, so please create an issue instead of a discussion.
Filing an issue suggests telling what version and render pipeline you're using. Without that info it's hard to help.

**Comment by artyomabgaryan** [comment]
done https://github.com/atteneder/glTFast/issues/549
